src/Components/DisplayActivity.js

```diff
@@ -0,0 +1,25 @@
+
+function DisplayActivity (props) {
+   if (props.activity !== undefined)
+
+      return (
+         <div className="activity">
+            { props.activity !== "" ?
```

Comment:
This counts as error handling to me! Okay, not quite for _errors_ but it's a good filter for results that come up empty.